docs/tutorial.qmd

```diff
@@ -0,0 +1,980 @@
+---
+title: "Using Flow"
+tbl-colwidths: [30,70]
+---
+
+## Core Concepts
+
+### Flow Type System
+
+Inspect Flow mirrors Inspect AI's object model with corresponding Flow types:
+
+-   **`FlowConfig`** — The top-level workflow definition that translates to an Inspect AI [Eval Set](https://inspect.aisi.org.uk/eval-sets.html). Contains a list of tasks and workflow settings.
```

Comment:
Should we say job definition instead of workflow definition? I'm not sure about mentioning Eval Set here. Or maybe should say something like "under the hood this uses Eval Set" with the caveat that it could change in the future.